### DISCUSSION TITLE
Getting started. Can't add any music providers

### DISCUSSION THREAD

**Discussion by EduMolinaIII** [music-assistant/support#1365] [Q&A] [answered]
Hi everybody!
I'm sorry if this has been already asked, but I can't find it in the discussions, nor the FAQ, nor the getting started page. I'm struggling to make music assistant work for me.
I can't find any way to add any music provider to my music assistant. Also, it doesn't appear in the sidebar of my HA.
I've tried to reinstall the plugin, many versions (beta and non beta). The only function I can see is the device menu, but not the provider menu.

What am I doing wrong? Thanks a lot in advance.

<img width="1280" alt="Screenshot 2023-07-12 at 09 10 51" src="https://github.com/music-assistant/hass-music-assistant/assets/128428864/e38c409e-60cf-4226-ba36-a6d250aba7ad">
<img width="1280" alt="Screenshot 2023-07-12 at 09 11 04" src="https://github.com/music-assistant/hass-music-assistant/assets/128428864/3ac1b038-0c7b-44b7-b30a-59a2a8feb129">
<img width="1280" alt="Screenshot 2023-07-12 at 09 11 11" src="https://github.com/music-assistant/hass-music-assistant/assets/128428864/75b7c921-ae0a-4597-8717-4ea156edb36b">
<img width="1280" alt="Screenshot 2023-07-12 at 09 11 37" src="https://github.com/music-assistant/hass-music-assistant/assets/128428864/bd993943-4487-4377-9675-5d6843dcdb70">

**Comment by OzGav** [comment, marked as answer]
I can see in the last image that you have the addon installed. Now click on "Ajustes" and then in there you will find the ability to add a provider

**Reply by EduMolinaIII** [reply]
Hi again! Thanks a lot for your fast response. I never search in the addon tab, just in the services > integrations > home assistant.

Thanks again!
